be/src/util/block_compression.cpp

```diff
@@ -86,6 +87,78 @@ class Lz4BlockCompression : public BlockCompressionCodec {
     size_t max_input_size() const override { return LZ4_MAX_INPUT_SIZE; }
 };
 
+class Lz4BlockCompression : public BlockCompressionCodec {
```

Comment:
What's the difference between LZ4 and LZ4Block?
LZ4 is a block algorithm default?